release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1278,6 +1278,26 @@ sourceStrategy:
+
+* Previously, installing a private (internal) {product-title} cluster on Azure Stack Hub failed because the installation program did not create the DNS record for the API VIP. This update removes the invalid check that caused this problem. The installation program now correctly creates DNS records for private cluster. link:https://bugzilla.redhat.com/show_bug.cgi?id=2061549[*BZ#2061549*]
+
+* Previously, uninstalling an IBM Cloud VPC cluster might have caused unexpected results. When a user uninstalled a cluster (cluster 1), the DNS records of another cluster (cluster 2) were removed when either the name of cluster 1 (example) was a subset of the name of cluster 2 (myexample) or both clusters shared a base domain. This update corrects this behavior. Only those resources specific to the cluster that is being uninstalled are now removed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2060617[*BZ#2060617*]
+
+* Previously, Azure Stack Hub did not support any disk types other than Standard_LRS. This update adds the ability to customize disk types, which allows clusters to have a default disk type with no manual customizations. This resulted in making the switch from hard coding the disk type to accepting the inputs from the user and validating it against the Stack Hub APIs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2061544[*BZ#2061544*])
+
+* Previously, when destroying a cluster, the ID of the private route5 hosted zone for the cluster was being incorrectly reported when the DNS records were deleted from the hosted zone. This caused incorrect host zone IDs to be reported in the log of the destroyer. This update uses the proper host zone ID in the log. As a result, the log  shows the correct hosted zone ID when destroying the DNS record in the base domain's hosted zone. (https://bugzilla.redhat.com/show_bug.cgi?id=1965969[*BZ#1965969*])
+
+* Previously, system proxy settings were not considered when requesting an AWS custom service endpoint. This update configures AWS custom service endpoint validation to consider the system proxy settings along with a HEAD request to the AWS custom service endpoint. As a result, the AWS custom service endpoint can be accessed from the user's machine. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2048451[*BZ#2048451*])
+
+* Previously, the installer used any Terraform provider in the `$PATH` on the installer host. Therefore, the installation would fail if there were Terraform providers in the `$PATH` that used an incorrect version or provider instead of the providers embedded in the installer. With this update, the installer embeds providers to a known directory and sets Terraform to use the known directory. As a result, installation will be successful because the installer will always use the providers in the known directory. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1932812[*BZ#1932812*])
```

Comment:
```suggestion
* Previously, the installer used any Terraform provider in the `$PATH` on the installer host. Therefore, the installation would fail if there were Terraform providers in the `$PATH` that used an incorrect version or provider instead of the providers embedded in the installation program. With this update, the installation program embeds providers to a known directory and sets Terraform to use the known directory. As a result, installation will be successful because the installation program will always use the providers in the known directory. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1932812[*BZ#1932812*])
```